app/views/boxes/show.haml

```diff
@@ -0,0 +1,45 @@
+- content_for(:subheader) do
+  .row.center.edit
+    .col.pe-10
+      .row
+        .col
+          %h2
+            = link_to boxes_path, class: "side-link", title: "Back" do
+              = image_tag "arrow-left.png"
+            Box detail
+
+.row
+  .col
+    .row
+      .col.pe-5
+        %label ID
+      .col
+        .value= @box.uuid
+
+    .row
+      .col.pe-5
+        %label Purpose
+      .col
+        .value= @box.purpose
+
+  .col.pe-5
+    .row
+      .col
+        = render "barcode_card"
+
+.row
+  .col
+    .row
+      .col.pe-3
+        %label Contents
+      .col.pe-9
+        - @samples.each do |sample|
+          .col.pe-7.batches-samples
+            .samples-row
+              .samples-item= sample.uuid
+              .sample-row-action= sample.batch.try(&:batch_number)
```

Comment:
This currently won't work with transferred samples because the `batch` relation gets removed on a transfer.
I'm thinking we should keep that reference to make things easier to reason about. And a sample always comes from its original batch, this can't really change. It shouldn't matter whether the sample is in the same location and institution as the batch it was created from.

Reply:
Yes, this is by design: until the box is transferred, we have the batch relationship, and can display the batch number to the sender, but once it's transferred, we lose the relationship, and the receiving institution won't see the batch number anymore.

Reply:
We shouldn't lose the data.  If I recall correctly, the relationship is removed, but the original batch ID is stored in another field of the Sample.
Whether the relationship needs to be removed or not, I don't know.  When we did that, it was thinking that it would be strange for the sample to belong to an institution and the batch to another.  But Johannes is making a good point, and I always prefer to try reflecting reality in the systems.  So, if it doesn't cause any colateral damage, I wouldn't disagree on keeping the relationship to the original batch.

Reply:
Even with keeping the relation we can choose to not show it when a sample is transferred. But it gives the easy option to show it when necessary.